src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService service = new UserService();
+        User userService = service.findByEmail(email);
+        if (userService != null && userService.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
you can simplify this code
```suggestion
        return userService != null && userService.getPassword().equals(password);         
```